components/profile/ProfileInfo.js

```diff
@@ -57,33 +59,34 @@ const ProfileInfo = ({ user }) => {
             </a>
           </div>
         )}
-        {user.twitterUsername && (
+        {user.websiteUrl && (
           <div className="flex items-center gap-1 mt-2 dark:text-gray-300">
-            <AiOutlineTwitter size={20} />
+            <BsLink45Deg size={22} />
             <a
-              href={`https://twitter.com/${user.twitterUsername}`}
+              href={`https://${user.websiteUrl}`}
               target="_blank"
               rel="noreferrer"
               className="hover:underline hover:text-gray-800 dark:hover:text-white"
             >
-              {"@"}
-              {user.twitterUsername}
+              {user.websiteUrl}
             </a>
           </div>
         )}
-        {user.websiteUrl && (
-          <div className="flex items-center gap-1 mt-2 dark:text-gray-300">
-            <BsLink45Deg size={22} />
-            <a
-              href={`https://${user.websiteUrl}`}
+
+        {user.socialAccounts?.nodes?.map((social)=>(
+          <div key={social.displayName} className="flex items-center gap-2 mt-2 dark:text-gray-300">
+            <Image alt='logo' width={20} height={20} src={`https://www.google.com/s2/favicons?sz=64&domain_url=${getApexDomain(social.url)}`}/>
```

Comment:
Please Don't use Next.js Image because vercel has a free quota of 1000 image optimization and it breaks the deployment after 1000 Image Next.js image optimizations

Reply:
Ok than can i use simple img

Reply:
Yeah